packages/now-node/src/types.ts

```diff
@@ -14,4 +14,5 @@ export type NowResponse = ServerResponse & {
   send: (body: any) => NowResponse;
   json: (jsonBody: any) => NowResponse;
   status: (statusCode: number) => NowResponse;
+  redirect: (statusCode: number, path: string) => NowResponse;
```

Comment:
```suggestion
  redirect: (statusCode: number, path: string) => NowResponse;
  redirect: (path: string) => NowResponse;
```

We need to support both method signatures here

Reply:
Ok @styfle  :D

Reply:
Also see the full implementation here: https://github.com/expressjs/express/blob/e1b45ebd050b6f06aa38cda5aaf0c21708b0c71e/lib/response.js#L909-L953